### ISSUE TITLE
Disable automatic image download

### ISSUE THREAD

**Issue by patoarvizu** [Neo11/ChitChat#12]
Is it possible to disable automatic image downloading? On my iOS device I have it setup where images are only downloaded when I tap on them but that doesn't happen on the desktop client.

Videos are however only downloaded when I tap on them (both on my device and on the desktop client) which is the intended setting.

Is there a way to get this behavior?

**Comment by c-o-m-m-a-n-d-e-r** [comment]
i don't know if this would be possible ...

**Comment by patoarvizu** [comment]
Is this an API/WhatsApp limitation? I'm not familiar with Objective-C so it' would be a bit hard for me to point to where a change could be made.

**Comment by Neo11** [comment]
I think WhatsApp must implement this feature because it isn't possible to limit the communication between WhatsApp Web and the App. We don't ask to get the messages, we only open web.whatsapp.com.

**Comment by c-o-m-m-a-n-d-e-r** [comment]
correct, but i will check the option to hook into the getMessage Event... 

need some time for that

with a little bit of luck the download is a separat call we can cancle before run.
this happens in the javascript part!
